sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -29,36 +29,10 @@ public FormRecognizerClientLiveTests(bool isAsync) : base(isAsync)
         {
         }
 
-        /// <summary>
-        /// Creates a <see cref="FormRecognizerClient" /> with the endpoint and API key provided via environment
-        /// variables and instruments it to make use of the Azure Core Test Framework functionalities.
-        /// </summary>
-        /// <param name="useTokenCredential">Whether or not to use a <see cref="TokenCredential"/> to authenticate. An <see cref="AzureKeyCredential"/> is used by default.</param>
-        /// <param name="apiKey">The API key to use for authentication. Defaults to <see cref="FormRecognizerTestEnvironment.ApiKey"/>.</param>
-        /// <returns>The instrumented <see cref="FormRecognizerClient" />.</returns>
-        private FormRecognizerClient CreateInstrumentedFormRecognizerClient(bool useTokenCredential = false, string apiKey = default)
-        {
-            var endpoint = new Uri(TestEnvironment.Endpoint);
-            var options = Recording.InstrumentClientOptions(new FormRecognizerClientOptions());
-            FormRecognizerClient client;
-
-            if (useTokenCredential)
-            {
-                client = new FormRecognizerClient(endpoint, TestEnvironment.Credential, options);
-            }
-            else
-            {
-                var credential = new AzureKeyCredential(apiKey ?? TestEnvironment.ApiKey);
-                client = new FormRecognizerClient(endpoint, credential, options);
-            }
-
-            return InstrumentClient(client);
-        }
-
         [Test]
         public void FormRecognizerClientCannotAuthenticateWithFakeApiKey()
         {
-            var client = CreateInstrumentedFormRecognizerClient(apiKey: "fakeKey");
+            var client = CreateFormRecognizerClient(apiKey: "fakeKey");
```

Comment:
Renaming this since we're now supporting creating a client without instrumenting (new parameter `skipInstrumenting` defaults to `false`).